cmd/cloudshell_open/main.go

```diff
@@ -549,3 +566,19 @@ func isSubPath(a, b string) (bool, error) {
 	}
 	return !strings.HasPrefix(v, ".."+string(os.PathSeparator)), nil
 }
+
+func instrumentlessCoupon() (*instrumentless.Coupon, error) {
+	ctx := context.TODO()
+
+	creds, err := transport.Creds(ctx)
+	if err != nil {
+		return nil, err
```

Comment:
do you mind wrapping the errors? they help with debugging  when users submit an error msg.